opm/material/viscositymodels/LBC.hpp

```diff
@@ -0,0 +1,213 @@
+            const Scalar v_c = FluidSystem::criticalVolume(compIdx);  // in m3/kmol
+            sumMm += x*Mm;
+            sumVolume += x*v_c;
+        }
+
+        LhsEval rho_pc = sumMm/sumVolume; //mixture pseudocritical density
+        LhsEval rho_r = rho/rho_pc;
+
+
+        LhsEval xsum_T_c = 0.0; //mixture pseudocritical temperature
+        LhsEval xsum_Mm = 0.0; //mixture molar mass
+        LhsEval xsum_p_ca = 0.0;  //mixture pseudocritical pressure
+        for (unsigned compIdx = 0; compIdx < FluidSystem::numComponents; ++compIdx) {
+            const Scalar& p_c = FluidSystem::criticalPressure(compIdx)/1e6; // in Mpa;
+            const Scalar& T_c = FluidSystem::criticalTemperature(compIdx);
+            const Scalar Mm = FluidSystem::molarMass(compIdx) * 1000; //in kg/kmol;
+            const auto& x = Opm::decay<LhsEval>(fluidState.moleFraction(phaseIdx, compIdx));
+            Scalar p_ca = p_c / MPa_atm;
+            xsum_T_c += x*T_c;
+            xsum_Mm += x*Mm;
+            xsum_p_ca += x*p_ca;
+        }
+        LhsEval zeta_tot = Opm::pow(xsum_T_c / (Opm::pow(xsum_Mm,3.0) * Opm::pow(xsum_p_ca,4.0)),1./6);
+
+        LhsEval my0 = 0.0;
+        LhsEval sumxrM = 0.0;
+        for (unsigned compIdx = 0; compIdx < FluidSystem::numComponents; ++compIdx) {
+            const Scalar& p_c = FluidSystem::criticalPressure(compIdx)/1e6; // in Mpa;
+            const Scalar& T_c = FluidSystem::criticalTemperature(compIdx);
+            const Scalar Mm = FluidSystem::molarMass(compIdx) * 1000; //in kg/kmol;
+            const auto& x = Opm::decay<LhsEval>(fluidState.moleFraction(phaseIdx, compIdx));
+            Scalar p_ca = p_c / MPa_atm;
+            Scalar zeta = std::pow(T_c / (std::pow(Mm,3.0) * std::pow(p_ca,4.0)),1./6);
+            LhsEval T_r = T/T_c;
+            LhsEval xrM = x * std::pow(Mm,0.5);
+            LhsEval mys = 0.0;
+            if (T_r <=1.5) {
+                mys = 34.0e-5*Opm::pow(T_r,0.94)/zeta;
+            } else {
+                mys = 17.78e-5*Opm::pow(4.58*T_r - 1.67, 0.625)/zeta;
+            }
+            my0 += xrM*mys;
+            sumxrM += xrM;
+        }
+        my0 /= sumxrM;
+
+        std::vector<Scalar> LBC = {0.10230,
+                                   0.023364,
+                                   0.058533,
+                                   -0.040758,  // trykkfeil i 1964-artikkel: -0.40758
+                                   0.0093324};
+
+        LhsEval sumLBC = 0.0;
+        for (int i = 0; i < 5; ++i) {
+            sumLBC += Opm::pow(rho_r,i)*LBC[i];
+        }
+
+        return (my0 + (Opm::pow(sumLBC,4.0) - 1e-4)/zeta_tot)/1e3; // mPas-> Pas
+    }
+
+
+        // Improved LBC model for CO2 rich mixtures. (Lansangan, Taylor, Smith & Kovarik - 1993)
+        template <class FluidState, class Params, class LhsEval = typename FluidState::Scalar>
+        static LhsEval LBCmodified(const FluidState& fluidState,
```

Comment:
If I did not see wrong, there are two `LBCModified` functions now, and one should be removed. 

And also, `LBCmodified` needs a more specific name.

Reply:
I will fix this